scripts/variantstore/wdl/GvsExtractCallset.wdl

```diff
@@ -283,11 +286,12 @@ task GetBQTableLastModifiedDatetime {
             gcloud auth activate-service-account --key-file=local.service_account.json
         fi
 
-        # bq needs the project name to be separate by a colon
-        DATASET_TABLE_COLON=$(echo ~{dataset_table} | sed 's/\./:/')
+        echo "project_id = ~{query_project}" > ~/.bigqueryrc
 
-        LASTMODIFIED=$(bq show  ~{"--project_id " + query_project} --location=US --format=json ${DATASET_TABLE_COLON} | python3 -c "import sys, json; print(json.load(sys.stdin)['lastModifiedTime']);")
+        # bq needs the project name to be separate by a colon
+        DATASET_TABLE_COLON=$(echo ~{fq_table} | sed 's/\./:/')
```

Comment:
just to confirm... this only replaces the first instance of a `.` correct, not all of them?

Reply:
correct. there is no '/g' at the end which means replace all of them.
tested on an extract of the 3 controls